models/section.schema.js

```diff
@@ -0,0 +1,32 @@
+const mongoose = require("mongoose");
+
+const sectionSchema = new mongoose.Schema(
+  {
+    _id: { type: String, required: true, unique: true },
+    name: { type: String, required: true, unique: true },
+    geofence_area: {
+      latitude: Number,
+      longitude: Number,
+    },
+    segments: [
+      {
+        id: { type: String, required: true, unique: true },
+        name: { type: String, required: true, unique: true },
+        geofence_area: {
+          latitude: Number,
+          longitude: Number,
+        },
+        deadline: Date,
+        volunteers: [
+          {
+            id: ObjectId,
```

Comment:
🚫 **[eslint]** <[no-undef](https://eslint.org/docs/rules/no-undef)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'ObjectId' is not defined.